src/test/java/com/twilio/http/NetworkHttpClientTest.java

```diff
@@ -156,6 +159,17 @@ public void testPost() throws IOException {
         assertEquals(resp.getContent(), "frobozz");
     }
 
+    @Test
+    public void testuserAgentExtensions() throws IOException {
+        setup(201, "frobozz", HttpMethod.POST, false);
+        mockRequestExtension = new Request(HttpMethod.GET, "http://testUrl");
+        mockRequestExtension.setUserAgentExtensions(Arrays.asList("ce-appointment-reminders/1.0.0", "code-exchange"));
+        Response resp = client.makeRequest(mockRequestExtension);
+
+        assertEquals(resp.getStatusCode(), 201);
+        assertEquals(resp.getContent(), "frobozz");
```

Comment:
Similar comment as the other test, there's no assertion the request builder (and thus the request) contains the extensions. If L134 of `NetworkHttpClient` was removed, this test would not fail.